lgc/test/CsBPermuteWave64.lgc

```diff
@@ -1,5 +1,6 @@
 ; NOTE: Assertions have been autogenerated by tool/update_llpc_test_checks.py UTC_ARGS: --tool lgc
 ; RUN: lgc -o - --mcpu=gfx1100 -filetype=asm %s | FileCheck -check-prefixes=CHECK %s
+; REQUIRES: do-not-run-me
```

Comment:
Disabling because the checks are auto-generated and will only work once the LLVM patch has landed.